getting_started.ipynb

```diff
@@ -0,0 +1,1962 @@
+    "    ]\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3f6682cc-576c-45f0-8b5b-52674a0f30a8",
+   "metadata": {},
+   "source": [
+    "This is how we have been using prompts, but now we will skip this step and invoke using the chain."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 59,
+   "id": "761b48c5-92e5-4857-b280-cdd486c29b02",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'In Japan, maguro (tuna) is the most popular sashimi. Globally, salmon sashimi tends to be more popular.'"
+      ]
+     },
+     "execution_count": 59,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "prompt_value = chat_prompt.format_messages(country=\"Japanese\", question=\"What is the most popular Sashimi in Japan vs the rest of the world?\")\n",
+    "\n",
+    "response = model.invoke(prompt_value)\n",
+    "response.content"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 60,
+   "id": "3893edc8-acc0-418e-ac5d-31a65b00b8b8",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "In Japan, the most popular sashimi is often Maguro (tuna), specifically the fatty part known as Otoro. Globally, Salmon sashimi tends to be more popular due to its rich flavor and widespread availability.\n"
+     ]
+    }
+   ],
+   "source": [
+    "from langchain_core.output_parsers import StrOutputParser\n",
+    "\n",
+    "chain = chat_prompt | model | StrOutputParser()\n",
+    "\n",
+    "print(\n",
+    "    chain.invoke(\n",
+    "        {\n",
+    "            \"country\": \"Japanese\", \n",
+    "            \"question\": \"What is the most popular Sashimi in Japan vs the rest of the world?\"\n",
+    "        }\n",
+    "    )\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3f186179-13e7-4056-9b62-9b4b1ac8de60",
+   "metadata": {},
+   "source": [
+    "### Wait in between API calls"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9f747f98-7b78-436f-a889-e3cff3f19bf9",
+   "metadata": {},
+   "source": [
+    "Depending on the users Azure plan and account will determine how many API requests can be made within a given interval. If too many requests are sent frequently an error may occur where the user will be told to wait **x** amount of time before sending another request.\n",
+    "\n",
+    "When creating a model, one of the parameters briefly mentioned earlier was the `max_retries` parameter. When calling the underlying Python OpenAI library, the library will wait and retry the call on your behalf at least 2 times (the default) before raising a RateLimiteError, or whatever `max retries` is set as.\n",
+    "\n",
+    "Otherwise, to ensure the user is not overwhelming the Azure services, make sure to wait between API calls when possible - this can be enforced by using the `time` package."
```

Comment:
I would probably drop this line - I think customers understanding that they will be throttled, and that this will generate errors, is probably sufficient.

Reply:
Agreed. I would instead link to this page about quotas and limits for customers to learn more: https://learn.microsoft.com/azure/ai-services/openai/quotas-limits

Reply:
I've changed this section to be at the top closer to where the max parameters is mentioned, I've also drop the 'wait' section and added a line with the link krista mentioned